File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int b, s, h;
    while(true)
    {
    printf("Height: ");
    b = get_int();
    if (b > -1 && b < 24)
        break;
    }
    
    
    for(int i = 0; i < b; i = i + 1)
    {
        s = b - 1 - i;
        h = b - s;
        
        
        for(int x = 0; x < s; x = x + 1)
            printf(" ");
        for(int x = 0; x < h; x = x + 1)
            printf("#");
        
        printf("  ");
        
        for(int x = 0; x < h; x = x + 1)
            printf("#");
            
        printf("\n");
        
        
    }
}

File: mario.py
import cs50

def main():
    while True:
        print ("Height: ", end="")
        b = cs50.get_int()
        if b > -1 and b < 24:
            break
        
    for i in range(b):
        s = b - 1 - i
        h = b - s
        
        for x in range(s):
            print (" ", end="")
        for x in range(h):
            print ("#", end="")
        
        print ("  ", end="")
        
        for x in range(h):
            print ("#", end="")
            
        print ("")
        
if __name__ == "__main__":
    main()